### DISCUSSION TITLE
umi4 css module 配置问题

### DISCUSSION THREAD

**Discussion by sakura-z-z** [umijs/umi#9984] [Q&A]
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/49021563/206132741-0d00528e-9119-4a4a-bb88-864c47235ffc.png">
umi4 可以正常启动， 但是commit 的时候会报这个问题。。。 配置如下

<img width="701" alt="image" src="https://user-images.githubusercontent.com/49021563/206133052-f910099e-94db-4e69-8e53-75c08eaacf23.png">
<img width="839" alt="image" src="https://user-images.githubusercontent.com/49021563/206133169-2a7b101b-a99f-4089-a5a9-11212466de58.png">
文件名中的 module  加上 删掉 没有区别

**Comment by sakura-z-z** [comment]
解决了。。。

**Reply by qiutian00** [reply]
> 解决了。。。

咋么解决的呢？
